lib/improver/tests/utilities/test_cli_utilities.py

```diff
@@ -64,11 +76,60 @@ def test_radii_and_lead(self):
     def test_both_used(self):
         """Tests if both arguments are used.
         The output is the same as if the second argument is None."""
```

Comment:
This doc-string is now wrong as you've gone for an exception rather than a warning. The exception messages are also rather cryptic (if viewed without all the context we have by virtue of working on this code). Can you make them clear, e.g.

```Neither radius or radii_by_lead_time have been set. One option should be specified.```
```Both radius and radii_by_lead_time have been set. Only one option should be specified.```